usrsctplib/netinet/sctp_output.c

```diff
@@ -14793,7 +14793,7 @@ sctp_lower_sosend(struct socket *so,
 out_unlocked:
 
 	if (local_soresv && stcb) {
-		atomic_subtract_int(&stcb->asoc.sb_send_resv, sndlen);
+		atomic_subtract_int(&stcb->asoc.sb_send_resv, (int32_t)sndlen);
```

Comment:
Fixed in https://github.com/sctplab/usrsctp/commit/078bd24ac1768bc6dc87d40b030434cba1987ea3.